doc/architecture.mmd

```diff
@@ -0,0 +1,64 @@
+flowchart LR
+    U("<img src='https://super.so/icon/dark/user.svg'; width='25'/>User")
+    subgraph Node1
+        FS1[(FileSystem)]
+        P1[Pod]
+        D1[Daemon]
+        R1[Recorder]
+        D1 -.->|install profile|FS1
+        P1 -.-> |use profile|FS1
+        D1 --> |gRPC: record profile| R1
+        R1 --> |record profile| P1
+        D1 -->|get/watch| P1
+    end
+    subgraph Node2
+        FS2[(FileSystem)]
+        P2[Pod]
+        D2[Daemon]
+        R2[Recorder]
+        D2 -.->|install profile|FS2
+        P2 -.-> |use profile|FS2
+        D2 --> |gRPC: record profile| R2
+        R2 --> |record profile| P2
+        D2 -->|get/watch| P2
+    end
```

Comment:
Do we want to use only 1 node for simplicity? The same would apply to everything `*2`.

Reply:
We can do that, just wanted to emphasis the multiplicity but are too many details.